Source/CarthageKit/Git.swift

```diff
@@ -57,6 +57,30 @@ public struct GitURL: Equatable {
 		}
 	}
 
+	/// Amount of time before a git repository is fetched again. Defaults to 1 minute
+	public static var fetchCacheInterval: NSTimeInterval = 60.0
```

Comment:
Yeah, it would have to be outside, but also not a global property that's _implicitly_ used everywhere. Instead, please make that explicit by making it a parameter to whatever code uses this.